spock-junit4/src/test/groovy/org/spockframework/junit4/junit/HandlingOfAssumptionViolatedException.groovy

```diff
@@ -99,7 +99,7 @@ def foo() {
 }
 
 def bar() {
-  expect: true
+  expect: false
```

Comment:
For consistency with the previous test, but used assertions worked also in the previous form.